docs/content/latest/releases/whats-new/latest-release.md

```diff
@@ -0,0 +1,87 @@
+---
+title: What's new in the v2.7 latest release series
+headerTitle: What's new in the v2.7 latest release series
+linkTitle: v2.7 (latest)
+description: Enhancements, changes, and resolved issues in the latest release series.
+headcontent: Features, enhancements, and resolved issues in the latest release series.
+image: /images/section_icons/quick_start/install.png
+aliases:
+  - /latest/releases/
+menu:
+  latest:
+    parent: whats-new
+    identifier: latest-release
+    weight: 2585
+isTocNested: true
+showAsideToc: true 
+---
+## v2.7.0 - Apr 19, 2021
+
+Yugabyte release 2.7.0 builds on our work in the 2.5 series, which fed into the 2.4 stable release. With release 2.7.0, we're planning development on a number of [new features](#new-features), as well as refinements to existing functionality.
+
+**Build:** `2.7.0.0-b17`
+
+### Downloads
+
+<a class="download-binary-link" href="https://downloads.yugabyte.com/yugabyte-2.7.0.0-darwin.tar.gz">
+  <button>
+    <i class="fab fa-apple"></i><span class="download-text">macOS</span>
+  </button>
+</a>
+&nbsp; &nbsp; &nbsp;
+<a class="download-binary-link" href="https://downloads.yugabyte.com/yugabyte-2.7.0.0-linux.tar.gz">
+  <button>
+    <i class="fab fa-linux"></i><span class="download-text">Linux</span>
+  </button>
+</a>
+<br />
+
+### Docker
+
+```sh
+docker pull yugabytedb/yugabyte:2.7.0.0-b17
+```
+
+### New Features
+
+#### Yugabyte Platform
+
+* Yugabyte Platform High Availability is now available in BETA. With this feature, you can deploy multiple platforms in an active-standby cluster, ensuring you can always monitor and manage your platform-managed universes regardless of outages to any particular platform in the HA cluster.
+* [[7225](https://github.com/yugabyte/yugabyte-db/issues/7225)] [[7228](https://github.com/yugabyte/yugabyte-db/issues/7228)] [UI] Enable the OpenShift tab, marked as beta
+
+#### Core Database
+
+* [[2717](https://github.com/yugabyte/yugabyte-db/issues/2717)] YSQL: Support ALTER FUNCTION command
+* [[6985](https://github.com/yugabyte/yugabyte-db/issues/6985)] YSQL: Add simple UNIQUE column
+
+### Improvements
+
+#### Yugabyte Platform
```

Comment:
Let me check with the team to confirm this list

Reply:
(The list is probably about to grow, since I'm going to add all the changes from the 2.5.4 branch that we didn't release.)